kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/utils/NamespaceUtils.java

```diff
@@ -23,6 +29,7 @@
 public class NamespaceUtils {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(NamespaceUtils.class);
+    private static final Map<CollectorElement, Set<String>> MAP_WITH_SUITE_NAMESPACES = new HashMap<>();
```

Comment:
Global variables .. ick.. not a good approach.    You need to store/identify the namespaces from which you want to scrape state, right?

Two ideas:

1.   How about labeling the namespaces with a unique id, and using that identify the set namespaces which you need to scrape?
2. Use a Store org.junit.jupiter.api.extension.ExtensionContext#getStore to cache the namespace names that the test uses, have TestLogCollector pull the list of namespaces from that.

Reply:
Opted for using `getStore`. Please review it